modules/perception/camera/app/obstacle_camera_perception.cc

```diff
@@ -183,21 +183,21 @@ void ObstacleCameraPerception::InitLane(
     const std::string &work_root,
     const app::PerceptionParam &perception_param) {
   // Init lane
-  CHECK(perception_param.has_lane_param()) << "Failed to include lane_param.";
-  {
+  CHECK_GT(perception_param.lane_param_size(), 0)
+      << "Failed to include lane_param";
+  for (int i = 0; i < perception_param.lane_param_size(); ++i) {
     // Initialize lane detector
-    auto lane_param = perception_param.lane_param();
+    const auto &lane_param = perception_param.lane_param(i);
     CHECK(lane_param.has_lane_detector_param())
         << "Failed to include lane_detector_param.";
     LaneDetectorInitOptions lane_detector_init_options;
```

Comment:
Please be cautious in using CHECK, and return gracefully when an exception happens.

Reply:
I agree! Sine this CHECK is in the initialization function and we must have it before running the code, We use CHECK here. 
Contrarily, We prevent to use CHECK in the process function.

Reply:
I know that we are still using CHECK functions in multiple process functions. We will remove them as many as possible from now on.

Reply:
Please do. Thank you!

Reply:
Will do! Thank you for the suggestion!

Reply:
@jinghaomiao, do you want me to fix CHECK in process functions in this PR or later?